content/guides/daily-tasks/netservices.md

```diff
@@ -5,10 +5,10 @@ date = "2010-05-11T15:29:32.000Z"
 tags = []
 +++
 
-The basic installation of Haiku contains several standard system network utilities including SSH and Telnet
+The basic installation of Haiku contains several standard system network services including a FTP, Telnet, and SSH Server.
```

Comment:
```suggestion
The basic installation of Haiku contains several standard system network services including an FTP, Telnet, and SSH server.
```

Reply:
"a FTP", otherwise looks great.

Reply:
reuploaded revision

Reply:
It actually is "an FTP server". With "a vs. an", it does not depend on how the following word is written (vowel -> an, consonant -> a), but how it's pronounced. "FTP" sounded out is "eff-tee-pee", starting with an "ee" sound, so it's "an FTP server".
Sorry to be such a nitpicker... :)

Reply:
I had no idea, thanks!